torch/_meta_registrations.py

```diff
@@ -3015,6 +3030,15 @@ def meta__foreach_unaop(self):
     return [torch.empty_like(s) for s in self]
 
 
+@register_meta_foreach({aten._foreach_abs.default: aten.abs.default})
+def meta__foreach_abs(self, _scalar_fn):
+    torch._check(
+        isinstance(self, List),
+        lambda: f"Expect List[Tensor] but got {type(self)}",
+    )
+    return [_scalar_fn(s) for s in self]
```

Comment:
Going fromt he name of the forach op to the aten op is trivial. Implement this generically and register it for all the foreach ops in one go. You can even get the list of forach ops from the OpInfos so that you don't need to write the whole list.

Reply:
in the previous PR you put together a list with all the Foreach OpInfos